server/webdriver_tests/screenshot/screenshot_test.py

```diff
@@ -28,13 +28,13 @@
         'height': 5500
     },
     {
-        'url': '/place?dcid=country/USA&topic=Demographics',
+        'url': '/place?dcid=country/USA?topic=Demographics',
```

Comment:
change here to /place/country/USA too?

Reply:
that was done on purpose, to test that the redirect works as expected